contentcuration/contentcuration/frontend/channelEdit/pages/StagingTreePage/index.vue

```diff
@@ -385,11 +392,21 @@
       },
     },
     watch: {
-      nodeId(newNodeId) {
-        this.elevated = false;
-        this.loadAncestors({ id: newNodeId });
-        this.loadChildren({ parent: newNodeId, root_id: this.stagingId });
-      },
+  nodeId(newNodeId) {
+    this.elevated = false;
+    this.isLoadingchild = true;
+
+    Promise.all([
+      this.loadAncestors({ id: newNodeId }),
+      this.loadChildren({ parent: newNodeId, root_id: this.stagingId }),
+      this.loadCurrentChannel({ staging: true }),
```

Comment:
Where did this `this.loadCurrentChannel({ staging: true }),` call come from? I don't see it in the previous code